ldaptor/test/test_distinguishedname.py

```diff
@@ -315,29 +315,38 @@ def testMalformed(self):
 
 class LDAPDistinguishedName_Prettify(unittest.TestCase):
     def testPrettifySpaces(self):
-        """str(DistinguishedName(...)) prettifies the DN by removing extra whitespace."""
+        """DistinguishedName(...).toWire() prettifies the DN by removing extra whitespace."""
         d=dn.DistinguishedName('cn=foo, o=bar,  c=us')
-        assert str(d) == 'cn=foo,o=bar,c=us'
+        assert d.toWire() == b'cn=foo,o=bar,c=us'
 
 class DistinguishedName_Init(unittest.TestCase):
-    def testString(self):
+    def testToWire(self):
         d=dn.DistinguishedName('dc=example,dc=com')
-        self.assertEqual(str(d), 'dc=example,dc=com')
+        self.assertEqual(d.toWire(), b'dc=example,dc=com')
 
     def testDN(self):
         proto=dn.DistinguishedName('dc=example,dc=com')
         d=dn.DistinguishedName(proto)
-        self.assertEqual(str(d), 'dc=example,dc=com')
+        self.assertEqual(d.toWire(), b'dc=example,dc=com')
+
+    def testEqualToByteString(self):
```

Comment:
I know that existing tests don't have docstring, but please try to add a docstring describing the scope/purpose of this test.

in this way, the test can be a documentation for a specific behavoiour of the equal function.

Reply:
Docstrings were added for ``testEqualToByteString`` and ``testEqualToString`` tests

Reply:
see https://jml.io/pages/test-docstrings.html for an excellent starting point for writing test docstrings

Reply:
Thanks! Made appropriate changes in docstrings, hope I got the point